### ISSUE TITLE
Compile error in export/distro/format.go

### ISSUE THREAD

**Issue by andresrinivasan** [edgexfoundry/edgex-go#474]
```
$ make docker
...
CGO_ENABLED=1 go build -ldflags "-X github.com/edgexfoundry/edgex-go.Version=0.6.1" -o cmd/export-distro/export-distro ./cmd/export-distro
# github.com/edgexfoundry/edgex-go/export/distro
export/distro/format.go:122:10: cannot assign 1 values to 2 variables
export/distro/format.go:128:21: cannot assign 1 values to 2 variables
```

**Comment by andresrinivasan** [comment]
PICNIC!

Used dep rather than glide.

**Comment by drasko** [comment]
@andresrinivasan at one point we (Mainflux team) added `dep`, PR was rejected. Let me try again, maybe we'll be better luck.

**Comment by jduranf** [comment]
Related with #539
